www/front_src/src/Navigation/Sidebar/Menu/index.tsx

```diff
@@ -91,11 +92,22 @@ const NavigationMenu = ({
     setCurrentTop(top);
     setHoveredIndex(index);
     setHoveredNavigationItemsDerived({ currentPage, levelName });
+    discardTimeout();
+  };
+
+  const discardTimeout = (): void => {
+    if (isNil(timeoutRef.current)) {
+      return;
+    }
+    clearTimeout(timeoutRef.current);
   };
 
   const handleLeave = (): void => {
-    setHoveredIndex(null);
-    setHoveredNavigationItems(null);
+    discardTimeout();
+    timeoutRef.current = setTimeout((): void => {
+      setHoveredIndex(null);
+      setHoveredNavigationItems(null);
+    }, 1000);
```

Comment:
You can use a duration from Material UI https://mui.com/material-ui/customization/default-theme/

Reply:
Yeah, I would use the `standard` duration (300ms) or `complex` duration (375ms)